dev-support/create-release/release-build.sh

```diff
@@ -235,12 +235,12 @@ if [[ "$1" == "publish-release" ]]; then
   $MVN versions:set -DnewVersion=$VERSION
   MAVEN_OPTS="${MAVEN_OPTS}" ${MVN} --settings $tmp_settings \
     clean install -DskipTests \
-    -Dcheckstyle.skip=true ${PUBLISH_PROFILES} \
+    -Dcheckstyle.skip=true "${PUBLISH_PROFILES}" \
     -Dmaven.repo.local="${tmp_repo}"
   pushd "${tmp_repo}/${groupid_as_dir}"
   # Remove any extra files generated during install
   # Remove extaneous files from module subdirs
-  find $modules -type f | grep -v \.jar | grep -v \.pom | xargs rm
+  find $modules -type f | grep -v \.jar | grep -v \.pom | xargs rm -rf
```

Comment:
nit: I think rm -f is good enough (-r not needed) since the find command already filters on -type f, you will only get the files.